test.py

```diff
@@ -322,5 +322,6 @@ def test_main(input_files, samplerate, niter, alpha, softmask, residual_model, m
     args, _ = parser.parse_known_args()
     args = inference_args(parser, args)
 
-    test_main(args.input, args.samplerate, args.niter, args.alpha, args.softmask, args.residual_model, args.model,
-              args.targets, args.outdir, args.no_cuda)
+    test_main(input_files=args.input, samplerate=args.samplerate, niter=args.niter, alpha=args.alpha,
+              softmask=args.softmask, residual_model=args.residual_model, model=args.model, targets=args.targets,
+              outdir=args.outdir, no_cuda=args.no_cuda)
```

Comment:
This looks good now, thanks. 

Regarding the indentation, your code pass PEP8 but for consistency reasons, I would prefer 

```
test_main(
    input_files=args.input, samplerate=args.samplerate, niter=args.niter, alpha=args.alpha,
    softmask=args.softmask, residual_model=args.residual_model, model=args.model, targets=args.targets,
    outdir=args.outdir, no_cuda=args.no_cuda
)
```

(my take on this is that this way, its easier to rename `test_main` without changing the indentation)